amr-wind/wind_energy/actuator/wing/fixed_wing_ops.H

```diff
@@ -45,6 +45,26 @@ struct ReadInputsOp<FixedWing, ActSrcLine>
         pp.queryarr("chord", wdata.chord_inp);
         bool use_fllc = false;
         pp.query("fllc", use_fllc);
+
+        std::string motion_type{"none"};
+        pp.query("motion_type", motion_type);
+        {
+            if (amrex::toLower(motion_type) == "none") {
+                wdata.motion_type = 0;
```

Comment:
Is there are reason to set these as integers as opposed to strings that pass the specified value directly?